mozdownload/scraper.py

```diff
@@ -541,6 +541,10 @@ def get_build_info_for_date(self, date, build_index=None):
                 build_index -= 1
                 if not build_index or self.is_build_dir(build):
                     break
+
+        if build_index >= len(parser.entries):
+            raise errors.NotFoundError('Specified build number is not found ', url)
```

Comment:
Lets sync with other failure messages which use `has not been`:

```suggestion
            raise errors.NotFoundError('Specified build number has not been found ', url)
```